pex/pip/foreign_platform.py

```diff
@@ -121,9 +121,6 @@ def patch(target):
         json.dump(patched_environment, markers_fp)
     env = dict(_PEX_PATCHED_MARKERS_FILE=markers_fp.name)
 
-    if isinstance(target, AbbreviatedPlatform):
-        args = tuple(iter_platform_args(target.platform, target.manylinux))
```

Comment:
Not worth a CI run to change, but we could get rid of the args variable and hardwire `args=()` below.

Reply:
Fixed, and then some.